torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -446,6 +446,12 @@ at::Tensor AtenXlaType::arange(at::Scalar start, at::Scalar end,
                         xla_options.get_scalar_type()));
 }
 
+at::Tensor& AtenXlaType::arange_out(at::Tensor& out, at::Scalar start,
```

Comment:
Why do we need this _out()?

Reply:
It's called from here: https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/TensorFactories.cpp#L74. At some point we might want to consider extending our code generation script to generate the `_out` versions, but until we need at least a handful of `_out` versions I wouldn't do it.

Reply:
If they are all dumb, we can add it to the generation. Or at least have a list of the ones to be generated, within gen.py, that we can control.
I'll take that task.

Reply:
I'm not sure how to *check* they're all dumb, which is the only concern I have regarding generating them. Manually writing tests for them would almost defeat the purpose of automatic generation since that's often harder than the operator itself for trivial ones. I'd rather hold on this.

Reply:
I mean, we whitelist the ones which are.
Adding to a python list is less spammy than polluting the code with same repeated stuff.